tests/Aspire.Confluent.Kafka.Tests/KafkaContainerFixture.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text;
+using System.Text.RegularExpressions;
+using Aspire.Components.Common.Tests;
+using Docker.DotNet.Models;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Configurations;
+using Testcontainers.Kafka;
+using Xunit;
+
+namespace Aspire.Confluent.Kafka.Tests;
+
+public partial class KafkaContainerFixture : IAsyncLifetime
+{
+    private sealed partial class ConfluentLocalKafkaBuilder : ContainerBuilder<ConfluentLocalKafkaBuilder, KafkaContainer, KafkaConfiguration>
+    {
+        public const string KafkaImage = "confluentinc/confluent-local:7.7.1";
```

Comment:
We should reuse the consts from KafkaContainerImageTags.

https://github.com/dotnet/aspire/blob/39912824d33de54be24baf842bf9228a6a84ac33/src/Aspire.Hosting.Kafka/KafkaContainerImageTags.cs#L12-L15

That way when they are updated, they are updated in the tests as well.

You can do this by source sharing into the test csproj, like this:

https://github.com/dotnet/aspire/blob/39912824d33de54be24baf842bf9228a6a84ac33/tests/Aspire.Hosting.Keycloak.Tests/Aspire.Hosting.Keycloak.Tests.csproj#L13

Reply:
done